src/Extensions/Logging/InitializationLogger.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Abstractions;
+using Microsoft.Omex.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>
+	/// InitStaticLogger is the logger to be used before the proper ILogger from DI is set
```

Comment:
```suggestion
	/// InitStaticLogger is the logger to be used before the proper ILogger from DI is set.
```

Reply:
Do we require a dot at the end of the comment?